File: delta/help.ts
import { Composer, Context } from "../deps.ts";
import * as start from "./start.ts";

const composer = new Composer();

export const message = `<b>Mavjud komandalar ro'yxati:</b>` +
  `\n` +
  `\n` +
  `/doc - <code>reply qilingan odamga dok borligi haqida eslatish</code>` +
  `\n` +
  `/useful - <code>rust haqida foydali yoki kerakli ma'lumotlar</code>` +
  `\n` +
  `/last - <code>eng oxirgi reliz haqida qisqacha ma'lumot</code>` +
  `\n` +
  `/version - <code>biron anniq reliz haqida to'liq ma'lumot</code>` +
  `\n` +
  `/off - <code>reply qilingan odamga offtop borligi haqida eslatish</code>` +
  `\n` +
  `/nometa - <code>to'g'ri savol berish haqida eslatma</code>` +
  `\n` +
  `/group - <code>rust ga oid guruh va hamjamiyatlar</code>` +
  `\n` +
  `/help - <code>ushbu xabarni qayta ko'rsatish</code>` +
  `\n` +
  `/about - <code>ushbu botimizning rivojlantirish qismi</code>` +
  `\n` +
  `/rules - <code>qoidalarni aks ettirish</code>` +
  `\n` +
  `/which - <code>ushbu guruh va foydalanuvchi metrik ma'lumotlarini ko'rsatish</code>`;
export const keyboard = start.keyboard;

composer.command("help", async (ctx: Context): Promise<void> => {
  await ctx.reply(message, {
    parse_mode: "HTML",
    reply_markup: keyboard,
  });
});

export default composer;

File: src/functions/help.rs
use super::start::keyboard;
use crate::bot::Command;
use orzklv::telegram::topic::Topics;
use teloxide::{prelude::*, types::ParseMode};

static TEXT: &[(&str, &str)] = &[
    ("help", "ushbu xabarni qayta ko'rsatish"),
    ("rules", "qoidalarni aks ettirish"),
    ("about", "ushbu botimizning rivojlantirish qismi"),
    ("group", "rust ga oid guruh va hamjamiyatlar"),
    ("roadmap", "boshlang'ich o'rganuvchilar uchun"),
    ("useful", "rust haqida foydali yoki kerakli ma'lumotlar"),
    ("latest", "eng oxirgi reliz haqida qisqacha ma'lumot"),
    ("version", "biron anniq reliz haqida to'liq ma'lumot"),
    ("warn", "mavzudan chetlashganga ogohlantiruv"),
];

pub async fn command(bot: &Bot, msg: &Message, _cmd: &Command) -> ResponseResult<()> {
    let mut text = String::new();

    text.push_str("<b>Mavjud komandalar ro'yxati:</b>\n\n");

    for cmd in TEXT {
        text.push('/');
        text.push_str(cmd.0);
        text.push_str(" - ");
        text.push_str(format!("<code>{text}</code>", text = cmd.1).as_str());
        text.push('\n');
    }

    bot.send_message_tf(msg.chat.id, text, msg)
        .parse_mode(ParseMode::Html)
        .reply_markup(keyboard())
        .await?;

    Ok(())
}
